frontend/src/components/SiteHeader.tsx

```diff
@@ -18,22 +18,22 @@ export default class SiteHeader extends React.Component<
         <Navbar>
           <Container>
             <Row>
-              <Link to="/">
-                <h1><img src="./placeholder_assets/logo.jpg" width="50" height="50" ></img>Buy</h1>
+              <Link style={{ textDecoration: 'none' }} to="/">
+                <h1 className="logo-img"><img src="./placeholder_assets/logo.jpg" width="50" height="50" ></img>Buy</h1>
```

Comment:
images should have `alt` text as per A11Y standards https://a11yproject.com/posts/alt-text/